src/spoke/BalanceSheet.sol

```diff
@@ -128,6 +128,22 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
         escrow_.authTransferTo(asset, tokenId, receiver, amount);
     }
 
+    /// @inheritdoc IBalanceSheet
+    function reserveIncrease(PoolId poolId, ShareClassId scId, address asset, uint256 tokenId, uint128 amount)
+        public
+        authOrManager(poolId)
+    {
+        escrow(poolId).reserveIncrease(scId, asset, tokenId, amount);
```

Comment:
Q: Isn't the name flipped? 🤔

Reply:
Yeah I thought this too, just matched the escrow naming. Not sure if @mustermeiszer had a specific reason for that.